tests/e2e/validator.go

```diff
@@ -41,6 +41,13 @@ type validator struct {
 	nodeKey          p2p.NodeKey
 }
 
+type account struct {
```

Comment:
I might differentiate this as a local account, but then again I suppose it's only accessible from the chain